pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -505,7 +505,7 @@ let
           revert = true;
         })
       ]
-      ++ lib.optionals (chromiumVersionAtLeast "131" && stdenv.hostPlatform.isAarch64) [
+      ++ lib.optionals (chromiumVersionAtLeast "132" && stdenv.hostPlatform.isAarch64) [
```

Comment:
The `chromiumVersionAtLeast` always returns true (becuse all our electron versions are at least 132), so it can be removed

Reply:
Additionally, I could add a comment like:
>TODO remove once electron_34 is EOL

Is that better?